tools/mason/MasonPublish.chpl

```diff
@@ -614,6 +644,23 @@ private proc moduleCheck(projectHome : string) throws {
   else return true;
 }
 
+/* Checks package for examples */
+proc exampleCheck(projectHome: string) {
+  if isDir(projectHome + '/example') {
+    const examples = listdir(projectHome + '/example');
+    if examples.size > 0 then return true;
```

Comment:
Sorry, didn't catch this earlier -- I suggest:

```chpl
return examples.size > 0;
```

Same applies to `testCheck`

Reply:
@ben-albrecht made the changes 👍